cmd/util.go

```diff
@@ -82,17 +83,59 @@ type kubeAuditReplicationControllers struct {
 }
 
 type Result struct {
-	err         int
-	namespace   string
-	name        string
-	capsAdded   []apiv1.Capability
-	imgName     string
-	capsDropped bool
-	kubeType    string
-	dsa         string
-	sa          string
-	token       *bool
-	imgTag      string
+	Err         int
```

Comment:
Is there any specific reason that you want to make these variable public instead of private?

Reply:
Yeah, afaik reflect only works on public variables, so to make the meta programming part work (that prints the correct fields of the `results` array) they needed to be exported.